packages/nextra-theme-docs/src/mdx-components.tsx

```diff
@@ -179,7 +179,7 @@ const Details = ({
 
   return (
     <details
-      className="my-4 rounded border border-gray-200 bg-white p-2 shadow-sm dark:border-neutral-800 dark:bg-neutral-900 first:mt-0 last:mb-0"
+      className="my-4 rounded border border-gray-200 bg-white p-2 shadow-sm dark:border-neutral-800 dark:bg-neutral-900"
```

Comment:
does there any reason to have `first:mt-0` class for details, ul, blockquote, table, p elements

Reply:
I think that makes it look better when there is nothing before the list and the title, e.g.:

```md

### title

<details>...</details>

```

Reply:
I think we can merge it and add tailwind typography (that was proposed in https://github.com/shuding/nextra/issues/16 issue) in another PR to fix https://github.com/shuding/nextra/issues/863